### DISCUSSION TITLE
Comments - So many to choose from - Which one do You prefer

### DISCUSSION THREAD

**Discussion by TheDoubleH** [HEIGE-PCloud/DoIt#380] [Q&A]
Ladies and Gentlemen -

DoIt offers support for so many different comment systems. Using disqus may be the easiest, given that will allow for a seamless migration if I ever decide to change theme, as it's natively supported by hugo. 

However, I am not sure I love the idea of getting adds in my comment feed. Curious to see what others use - and what it took to get it to work. I am in particular curious about the systems that use github issues.

**Comment by HEIGE-PCloud** [comment]
[utterances](https://utteranc.es/) and [Gitalk](https://gitalk.github.io/) use GitHub issues to host the comments.
[giscus](https://giscus.app/) uses GitHub discussion.

There are... not really much difference. utterances and giscus may be a little bit easier to set up.
